Classes/CRON/FormBuilder/Controller/FormBuilderController.php

```diff
@@ -130,6 +134,37 @@ protected function createMailData($node, $data) {
 	}
 
 
+
+	/**
+	 * Checks the HMAC for the form submitted data
+	 */
+	protected function checkHmac() {
+
+		$trustedProperties = $this->request->getInternalArgument('__trustedProperties');
+
+		if ($trustedProperties) {
+
+			if ($this->request->hasArgument('data')
```

Comment:
Aufgrund des Renderings der Formelemente in einem anderen Kontexts als f:form, muss man das selbst machen. Das   geht sonst leider nicht anders.